.gitignore

```diff
@@ -24,4 +24,6 @@ data/ipfs/
 dist/
 .idea/
 # ignore settings file as it could contain secrets
-settings.yaml
\ No newline at end of file
+settings.yaml
+
+test.json
```

Comment:
What's this?

Reply:
It gets generated when you run all tests using the command used in the github actions